### ISSUE TITLE
Enqueue button no longer working

### ISSUE THREAD

**Issue by tkittich** [ArtVentureX/sd-webui-agent-scheduler#38]
Not sure if I did something wrong, but the Enqueue button no longer working is no longer working in Vlad's SD. Nothing happens after clicking the Enqueue. Nothing is added to the queue. 

I am using the latest versions.

**Comment by shlomitgueta** [comment]
mine too :(

**Comment by artventuredev** [comment]
Please provide some information for the debugging:
- SD webui version, commit hash
- Logs from SD console, if using Vlad fork please run with `--debug` flag
- Any error message on browser console: press F12, open Console tab, refresh browser to see the logs

**Comment by shlomitgueta** [comment]
> Please provide some information for the debugging:
> 
> * SD webui version, commit hash
> * Logs from SD console, if using Vlad fork please run with `--debug` flag
> * Any error message on browser console: press F12, open Console tab, refresh browser to see the logs



12:14:43-988795 INFO     Python 3.10.6 on Windows
12:14:44-032739 INFO     Version: b340e3f4 Sun Jun 18 16:30:59 2023 -0400
12:14:44-366063 INFO     Latest published version: 9a7c765506efef5d77fca7756df99f98dc218497 2023-06-23T08:08:11Z


sorry,but were to put this coman:--debug? i am using vlad.
also cmd window not showing any errors

**Comment by artventuredev** [comment]
> sorry,but were to put this coman:--debug? i am using vlad.

Instead of double click the `webui.bat` file, open it using the command prompt:
- Open Command prompt
- Drag the `webui.bat` file into the command prompt
- Add `--debug` to the and (remember to add a space first), then hit Enter

And also for the browser:
- Press F12, open `Console` tab
- Refresh the browser F5 or Ctrl + R
- Watch for any error log in the `Console` tab

**Comment by shlomitgueta** [comment]
> > sorry,but were to put this coman:--debug? i am using vlad.
> 
> Instead of double click the `webui.bat` file, open it using the command prompt:
> 
> * Open Command prompt: Press Win + R and type cmd then Enter, or search for Command Prompt in the start menu
> * Drag the `webui.bat` file into the command prompt
> * Add `--debug` to the end (remember to add a space first), then hit Enter
> 
> And also for the browser:
> 
> * Press F12, open `Console` tab
> * Refresh the browser F5 or Ctrl + R
> * Watch for any error log in the `Console` tab

in the consol:
VM3789:3 Uncaught (in promise) ReferenceError: submit_enqueue is not defined
    at Object.eval [as frontend_fn] (eval at <anonymous> (Blocks.svelte:67:25), <anonymous>:3:14)
    at je (Blocks.svelte:277:6)
eval @ VM3789:3
je @ Blocks.svelte:277
Promise.then (async)
je @ Blocks.svelte:282
await in je (async)
(anonymous) @ Blocks.svelte:373
(anonymous) @ index.mjs:1170
Fr @ index.mjs:1170
f @ Button.svelte:12
(anonymous) @ index.mjs:1170
Fr @ index.mjs:1170
v @ Button.svelte:11
Show 4 more frames

**Comment by tkittich** [comment]
> Please provide some information for the debugging:
> 
> * SD webui version, commit hash, extension commit hash (should be available in the extensions tab)
> * Logs from SD console, if using Vlad fork please run with `--debug` flag
> * Any error message on browser console: press F12, open Console tab, refresh browser to see the logs

Here's my SD console log
```
PS D:\theera\aiart\automatic> .\webui.bat --debug
Using VENV: D:\theera\aiart\automatic\venv
17:17:21-952591 INFO     Starting SD.Next
17:17:21-957101 INFO     Python 3.10.11 on Windows
17:17:21-992644 INFO     Version: b80fcca9 Thu Jun 22 13:05:26 2023 -0400
17:17:22-522312 INFO     Latest published version:
                         9a7c765506efef5d77fca7756df99f98dc218497 2023-06-23T08:08:11Z
17:17:22-527320 DEBUG    Setting environment tuning
17:17:22-527320 DEBUG    Torch overrides: cuda=False rocm=False ipex=False diml=False
17:17:22-527320 DEBUG    Torch allowed: cuda=True rocm=True ipex=True diml=True
17:17:22-529823 INFO     nVidia CUDA toolkit detected
17:17:24-302564 INFO     Torch 2.0.1+cu118
17:17:24-312574 INFO     Torch backend: nVidia CUDA 11.8 cuDNN 8700
17:17:24-312574 INFO     Torch detected GPU: NVIDIA GeForce RTX 3090 VRAM 24576 Arch (8,
                         6) Cores 82
17:17:24-444007 DEBUG    Repository update time: Fri Jun 23 00:05:26 2023
17:17:24-446009 DEBUG    Previous setup time: Fri Jun 23 07:23:18 2023
17:17:24-447010 INFO     Disabled extensions-builtin:
                         ['stable-diffusion-webui-images-browser']
17:17:24-448010 INFO     Enabled extensions-builtin: ['a1111-sd-webui-lycoris',
                         'clip-interrogator-ext', 'LDSR', 'Lora',
                         'multidiffusion-upscaler-for-automatic1111', 'ScuNET',
                         'sd-dynamic-thresholding', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet',
                         'stable-diffusion-webui-rembg', 'SwinIR']
17:17:24-453535 INFO     Disabled extensions: ['stable-diffusion-webui-images-browser']
17:17:24-454535 INFO     Enabled extensions: []
17:17:24-455536 DEBUG    Latest extensions time: Fri Jun 23 11:25:25 2023
17:17:24-455536 DEBUG    Timestamps: version:1687453526 setup:1687479798
                         extension:1687494325
17:17:24-456537 INFO     Verifying requirements
17:17:24-467593 INFO     Verifying packages
17:17:24-467593 INFO     Installing repositories
17:17:28-782206 INFO     Installing submodules
17:17:29-972417 DEBUG    Submodules list: 025dea96720197dd4486a5bb8e2f4d72a95a3088
                         extensions-builtin/a1111-sd-webui-lycoris (heads/main)
                          4d865c5476813d304b1321c25dea8ddb1844d1bd
                         extensions-builtin/clip-interrogator-ext (heads/main)
                          de4888103c03d88c0d32af6d453373b47291687a
                         extensions-builtin/multidiffusion-upscaler-for-automatic1111
                         (sd-webui-v1.1.0-30-gde48881)
                          fa12a88ea071d83c654a2bc67c05fef5c5f3bb0f
                         extensions-builtin/sd-dynamic-thresholding (heads/master)
                          14d5b61ae776ed7a19140deea4189ce8a105bd4b
                         extensions-builtin/sd-extension-system-info (heads/main)
                         +c291dc56435e0b8634313d0ae57cd276c9040187
                         extensions-builtin/sd-webui-agent-scheduler (heads/main)
                         +784eadbba6600b3ecc2e3b07363895e42d6b3e8f
                         extensions-builtin/sd-webui-controlnet (heads/main)
                          b2f6e4cbe507078e3bf2b9d5fce6f83e2e233aa8
                         extensions-builtin/stable-diffusion-webui-images-browser
                         (heads/main)
                          3d9eedbbf0d585207f97d5b21e42f32c0042df70
                         extensions-builtin/stable-diffusion-webui-rembg (heads/master)
                          c7fd336c5d5c0f05378dd34e7b5ca6ac8e972773 modules/lora
                         (v0.6.4-121-gc7fd336)
                          57d71913bf47a48dc7dd519b89dd30e09c937b83 modules/lycoris
                         (heads/main)
                          f941746c0ed2afcaa37c1ec77b86da4dee131bee wiki (heads/master)
17:17:30-574419 INFO     Updating submodules
17:17:40-822248 DEBUG    Installed packages: 215
17:17:40-822248 DEBUG    Extensions all: ['a1111-sd-webui-lycoris',
                         'clip-interrogator-ext', 'LDSR', 'Lora',
                         'multidiffusion-upscaler-for-automatic1111', 'ScuNET',
                         'sd-dynamic-thresholding', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet',
                         'stable-diffusion-webui-rembg', 'SwinIR']
17:17:42-608438 DEBUG    Running extension installer:
                         D:\theera\aiart\automatic\extensions-builtin\clip-interrogator-ex
                         t\install.py
17:17:52-112048 DEBUG    Running extension installer:
                         D:\theera\aiart\automatic\extensions-builtin\sd-extension-system-
                         info\install.py
17:17:53-403047 DEBUG    Running extension installer:
                         D:\theera\aiart\automatic\extensions-builtin\sd-webui-agent-sched
                         uler\install.py
17:17:54-760760 DEBUG    Running extension installer:
                         D:\theera\aiart\automatic\extensions-builtin\sd-webui-controlnet\
                         install.py
17:17:55-972008 DEBUG    Running extension installer:
                         D:\theera\aiart\automatic\extensions-builtin\stable-diffusion-web
                         ui-rembg\install.py
17:17:56-453843 DEBUG    Extensions all: []
17:17:56-453843 INFO     Extensions enabled: ['a1111-sd-webui-lycoris',
                         'clip-interrogator-ext', 'LDSR', 'Lora',
                         'multidiffusion-upscaler-for-automatic1111', 'ScuNET',
                         'sd-dynamic-thresholding', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet',
                         'stable-diffusion-webui-rembg', 'SwinIR']
17:17:56-461854 INFO     Updating Wiki
17:17:57-322057 DEBUG    Setup complete without errors: 1687515477
17:17:57-332075 INFO     Extension preload: 0.0s
                         D:\theera\aiart\automatic\extensions-builtin
17:17:57-332075 INFO     Extension preload: 0.0s D:\theera\aiart\automatic\extensions
17:17:57-342170 DEBUG    Memory used: 0.33 total: 63.92 Collected 0
17:17:57-342170 DEBUG    Starting module: <module 'webui' from
                         'D:\\theera\\aiart\\automatic\\webui.py'>
17:17:57-342170 INFO     Server arguments: ['--debug']
17:17:57-352045 DEBUG    Loading Torch
17:18:00-412168 DEBUG    Loading Gradio
17:18:01-451886 DEBUG    Loading Modules
17:18:03-155600 INFO     Libraries loaded
17:18:03-158110 DEBUG    Entering start sequence
17:18:03-245953 DEBUG    Version: {'app': 'sd.next', 'updated': '2023-06-22', 'hash':
                         'b80fcca9', 'url':
                         'https://github.com/vladmandic/automatic/tree/master'}
17:18:03-247955 INFO     Using data path: D:\theera\aiart\automatic
17:18:03-249957 DEBUG    Event loop: <_WindowsSelectorEventLoop running=False closed=False
                         debug=False>
17:18:03-250957 DEBUG    Entering initialize
17:18:03-253083 INFO     Available VAEs: D:\theera\aiart\aiart\models\VAE 5
17:18:03-302110 INFO     Available models: D:\theera\aiart\aiart\models\Stable-diffusion
                         45
17:18:03-389984 DEBUG    Loading scripts
17:18:05-402104 INFO     ControlNet v1.1.224
ControlNet v1.1.224
ControlNet preprocessor location: D:\theera\aiart\automatic\extensions-builtin\sd-webui-controlnet\annotator\downloads
17:18:05-561184 INFO     ControlNet v1.1.224
ControlNet v1.1.224
17:18:06-492031 DEBUG    Scripts load: ['a1111-sd-webui-lycoris:1.089s',
                         'clip-interrogator-ext:0.111s', 'Lora:0.157s',
                         'sd-extension-system-info:0.05s',
                         'sd-webui-agent-scheduler:0.31s', 'sd-webui-controlnet:0.366s',
                         'stable-diffusion-webui-rembg:0.86s']
Scripts load: ['a1111-sd-webui-lycoris:1.089s', 'clip-interrogator-ext:0.111s', 'Lora:0.157s', 'sd-extension-system-info:0.05s', 'sd-webui-agent-scheduler:0.31s', 'sd-webui-controlnet:0.366s', 'stable-diffusion-webui-rembg:0.86s']
17:18:06-621855 INFO     Loading UI theme: name=gradio/default style=Auto
17:18:06-635097 DEBUG    Creating UI
17:18:06-637100 WARNING  Triggered lyco-patch-lora, will take lora_dir and <lora> format.
17:18:07-012914 DEBUG    Extensions list loaded:
                         D:\theera\aiart\automatic\html\extensions.json
Running on local URL:  http://127.0.0.1:7860
17:18:08-193618 INFO     Local URL: http://127.0.0.1:7860/
17:18:08-193618 DEBUG    Gradio registered functions: 1137
17:18:08-193618 INFO     Initializing middleware
17:18:08-201622 DEBUG    Creating API
17:18:08-414817 DEBUG    Scripts setup: ['Tiled Diffusion:0.014s', 'Tiled VAE:0.008s',
                         'ControlNet:0.034s', 'Alternative:0.009s']
17:18:08-416320 DEBUG    Scripts components: []
17:18:08-593037 INFO     Model metadata saved: D:\theera\aiart\automatic\metadata.json 8
17:18:08-593037 DEBUG    Model auto load disabled
17:18:08-593037 INFO     Startup time: 11.1s (torch=3.1s gradio=1.0s libraries=1.7s
                         vae=0.1s codeformer=0.1s scripts=3.1s onchange=0.1s ui=1.2s
                         launch=0.4s app-started=0.2s)
17:20:00-382789 DEBUG    Server alive=True Requests=77 memory used: 1.01 total: 63.92
```

and browser console log
```
:7860/openpose_editor_index:1     Failed to load resource: the server responded with a status of 404 (Not Found)
extension.mjs:1 Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec.
set-hints.js?1687447219.4215946:84 set-hints Object
VM437:3 Uncaught (in promise) ReferenceError: submit_enqueue is not defined
    at Object.eval [as frontend_fn] (eval at <anonymous> (Blocks.svelte:67:25), <anonymous>:3:14)
    at je (Blocks.svelte:277:6)
eval @ VM437:3
je @ Blocks.svelte:277
Promise.then (async)
je @ Blocks.svelte:282
await in je (async)
(anonymous) @ Blocks.svelte:373
(anonymous) @ index.mjs:1170
Fr @ index.mjs:1170
f @ Button.svelte:12
(anonymous) @ index.mjs:1170
Fr @ index.mjs:1170
v @ Button.svelte:11
Show 4 more frames
```

**Comment by artventuredev** [comment]
Please try the fix in this comment: https://github.com/ArtVentureX/sd-webui-agent-scheduler/issues/4#issuecomment-1575986274

**Comment by tkittich** [comment]
> Please try the fix in this comment: [#4 (comment)](https://github.com/ArtVentureX/sd-webui-agent-scheduler/issues/4#issuecomment-1575986274)

The registry edit fixed the problem. Thanks a lot. ^^
